packages/core/src/utils.js

```diff
@@ -573,7 +573,11 @@ export function optionsList(schema) {
     return altSchemas.map((schema, i) => {
       const value = toConstant(schema);
       const label = schema.title || String(value);
-      return { label, value };
+      return {
+        ...schema,
```

Comment:
Actually, I was just thinking it might be better to actually explicitly include the schema in a `schema` keyword. I know that `label` and `value` currently aren't reserved keywords in JSON Schema, but this way, we can prevent further conflicts in case it happens in the future, and don't have to worry about `label` and `value` potentially overriding something within the schema. What do you think?

```suggestion
        schema,
```

Reply:
agreed here, this makes sense to me, pushed a commit with this change made